spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/SubscriberToApiSubscriberSpec.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace spec\Akeneo\Bundle\FrontofficeBundle\Form\EventSubscriber;
+
+use Akeneo\Bundle\FrontofficeBundle\Security\Core\User\OAuthUser;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+class EventToApiSubscriberSpec extends ObjectBehavior
```

Comment:
Bad class name.
Moreover, we should rename `Subscriber` entity to something less misleading like `Subscription`